pkg/controller/plan/kubevirt.go

```diff
@@ -636,11 +636,51 @@ func (r *KubeVirt) createVolumesForOvirt(vm ref.Ref) (err error) {
 		if err != nil {
 			return
 		}
+		err = r.createPodToBindPVC(vm, pvc.Name)
```

Comment:
it's not really part of the creation of the volumes.. maybe a better place would be `Execute` in migration.go?